format/src/main/thrift/schema.thrift

```diff
@@ -124,6 +124,7 @@ struct TableSchema{
 	1: required string table_id;  // ID used to
 	2: required list<ColumnSchema> table_columns; // Columns in the table
 	3: required SchemaEvolution schema_evolution; // History of schema evolution of this table
+	4: optional i32 block_size
```

Comment:
@Zhangshunyu ...I agree with @jackylk ...just like we are storing the table path in metastore, same way block size can also be stored as it is metadata for the table. schema file we write should only describe the structure the table and not the metadata for table